Makefile

```diff
@@ -209,6 +212,9 @@ cuda-gen.cu    := $(sort $(wildcard backends/cuda-gen/*.cu))
 occa.c         := $(sort $(wildcard backends/occa/*.c))
 magma.c        := $(sort $(wildcard backends/magma/*.c))
 magma.cu       := $(sort $(wildcard backends/magma/*.cu))
+hip.hip        := $(sort $(wildcard backends/hip/*.hip))
+hip.cpp        := $(sort $(wildcard backends/hip/*.cpp))
```

Comment:
Do we need `.hip`? I thought they recommend `*.hip.cpp`, which would keep editor features working.

Reply:
Good point.  I originally thought it was cleaner to separate HIP files from the other `*.cpp` files this way, but I think the change in the Makefile for cfb6011c9806ecc9346fa1da351e922dcf03717b will work while being acceptably clear (?)